_inc/client/state/setup-wizard/actions.js

```diff
@@ -34,11 +37,35 @@ export const updateSetupWizardQuestionnaire = answer => {
 export const saveSetupWizardQuestionnnaire = () => {
 	return ( dispatch, getState ) => {
 		dispatch( { type: JETPACK_SETUP_WIZARD_QUESTIONNAIRE_UPDATE } );
+
+		const setupWizard = getState().jetpack.setupWizard;
+
 		return restApi
-			.saveSetupQuestionnaire( getState().jetpack.setupWizard.questionnaire )
+			.saveSetupQuestionnaire( {
+				questionnaire: setupWizard.questionnaire,
+				status: setupWizard.status,
+			} )
 			.then( () => {
 				dispatch( { type: JETPACK_SETUP_WIZARD_QUESTIONNAIRE_SAVE_SUCCESS } );
 			} )
 			.catch( error => dispatch( { type: JETPACK_SETUP_WIZARD_QUESTIONNAIRE_SAVE_FAIL, error } ) );
 	};
 };
+
+export const updateSetupWizardStatus = status => {
+	return ( dispatch, getState ) => {
+		dispatch( { type: JETPACK_SETUP_WIZARD_STATUS_UPDATE, status } );
+
+		const setupWizard = getState().jetpack.setupWizard;
+		return restApi
+			.saveSetupQuestionnaire( {
+				status: setupWizard.status,
+			} )
+			.then( () => {
+				dispatch( { type: JETPACK_SETUP_WIZARD_QUESTIONNAIRE_UPDATE_SUCCESS } );
+			} )
+			.catch( error =>
+				dispatch( { type: JETPACK_SETUP_WIZARD_QUESTIONNAIRE_UPDATE_FAIL, error } )
```

Comment:
I see the error in console when visiting `setup` page:

```
VM1343:1 Uncaught (in promise) Error: Actions may not have an undefined "type" property. Have you misspelled a constant?
    at Object.performAction (<anonymous>:1:34581)
    at y (<anonymous>:1:36743)
    at e (<anonymous>:1:40562)
    at index.js:11
    at dispatch (redux.js:638)
    at actions.js:68
```

It seems that both `JETPACK_SETUP_WIZARD_QUESTIONNAIRE_UPDATE_*` not defined in `state/action-types`

Reply:
Also, is it expected that `STATUS_UPDATE` is used together with `QUESTIONNAIRE_UPDATE_*` actions?

Reply:
This was an error, they were supposed to use `JETPACK_SETUP_WIZARD_STATUS_UPDATE_SUCCESS` etc but I had mistyped QUESTIONNAIRE_UPDATE.